torch/testing/_internal/common_methods_invocations.py

```diff
@@ -274,15 +274,27 @@ def sample_inputs_as_strided_scatter(op_info, device, dtype, requires_grad, **kw
         ((1,), (1,), (1,), 0),
         ((3, 3), (2, 2), (1, 2), 0),
         ((3, 3), (2, 2), (1, 2), 1),
-        ((16,), (2, 2, 2, 2), (1, 1, 1, 1), 0),
-        ((16,), (2, 1, 1, 2), (1, 7, 7, 1), 0),
+        ((3, 3), (2, 2), (2, 1), 0),
+        ((16,), (2, 2, 2, 2), (8, 4, 2, 1), 0),
```

Comment:
What's the change here? Should the test cases have a comment describing the property they assume?

Reply:
added a comment above, the removed code was causing non-deterministic behaviour so I modified the strides